packages/artillery-plugin-expect/test/run.sh

```diff
@@ -17,7 +17,7 @@ cleanup() {
 
 trap cleanup EXIT
 
-docker run --name mockingjay -p 9090:9090 -v "$DIR":/data "quii/mockingjay-server:$MOCKINGJAY_VERSION" --config /data/mock-pets-server.yaml &
+docker run --name mockingjay -p 9090:9099 -v "$DIR":/data "quii/mockingjay-server:$MOCKINGJAY_VERSION" --config /data/mock-pets-server.yaml &
```

Comment:
I assume this is a mapping `X:Y` but correct me if I'm wrong.

Reply:
other way around, it's host port first, container port second

Reply:
https://docs.docker.com/engine/reference/commandline/run/#publish